src/overlays/actors/ovl_En_Mt_tag/z_en_mt_tag.c

```diff
@@ -440,7 +440,8 @@ void EnMttag_PotentiallyRestartRace(EnMttag* this, PlayState* play) {
             CLEAR_EVENTINF(EVENTINF_11);
             CLEAR_EVENTINF(EVENTINF_12);
             CLEAR_EVENTINF(EVENTINF_13);
-            gSaveContext.eventInf[2] = ((gSaveContext.eventInf[2] & 0xF) + 1) | (gSaveContext.eventInf[2] & 0xF0);
+            gSaveContext.eventInf[EVENTINF_02] =
+                ((gSaveContext.eventInf[EVENTINF_02] & 0xF) + 1) | (gSaveContext.eventInf[EVENTINF_02] & 0xF0);
```

Comment:
and here

Reply:
Well, this tells us that this *isn't* packed in the same way as other eventInf things. It's showing that it's packing two four-bit values into a single byte, similar to the dog text ID offset stuff.